src/gen/gen_mutations.go

```diff
@@ -35,6 +35,23 @@ func allowExtendedLastParamMutatorHelper() func(ScrapeResult) ScrapeResult {
 	}
 }
 
+func allowOptionalLastParamMutatorHelper() func(ScrapeResult) ScrapeResult {
```

Comment:
There's a small improvement we can do it here. I'll raise a small change to your PR later.